File: src/test/testData/java/cccc/MyClass.java
public class MyClass {
    class MyInnerClass {
        public String innerGreeting() {
            return "Hello, inner!";
        }

        public String innerFarewell() {
            return "Hello, inner farewell!";
        }
    }

    public String greeting() {
        return "Hello, world!";
    }

    static class MyStaticInnerClass {
        public String innerGreeting() {
            return "Hello, inner!";
        }

        public String innerFarewell() {
            return "Hello, inner farewell!";
        }
    }

    public String farewell() {
        return "Farewell, cruel world!";
    }
}

File: src/test/testData/kotlin/cccc/MyClass.kt
class MyClass {
    internal inner class MyInnerClass {
        fun innerGreeting(): String {
            return "Hello, inner!"
        }

        fun innerFarewell(): String {
            return "Hello, inner farewell!"
        }
    }

    fun greeting(): String {
        return "Hello, world!"
    }

    internal class MyStaticInnerClass {
        fun innerGreeting(): String {
            return "Hello, inner!"
        }

        fun innerFarewell(): String {
            return "Hello, inner farewell!"
        }
    }

    fun farewell(): String {
        return "Farewell, cruel world!"
    }
}